firefox-ios/Client/Frontend/Browser/Tabs/Middleware/RemoteTabsPanelMiddleware.swift

```diff
@@ -67,23 +71,37 @@ class RemoteTabsPanelMiddleware {
                                                actionType: RemoteTabsPanelActionType.refreshDidBegin)
             store.dispatch(action)
 
-            getRemoteTabs(window: window)
+            getTabsAndDevices(window: window, useCache: useCache)
```

Comment:
@linabutler I want to call out what I did here in the event that it's not right. I saw that the old `getRemoteTabs` function was dispatching a `refreshDidSucceed` action and I didn't want to send another so before that so I condensed the calls.

If it makes more sense to have two separate actions no worries, I just thought it'd get messy retrieving the devices and client and tabs data both times.